### PR TITLE
Type enumeration and uint converter

### PR DESCRIPTION
**Pull request by nohkwak**
- Add Klaytn Tx & AccountKey type enumeration 
- Update existing examples using this enumeration
- Add Klaytn unit (pep, kpep, Mpep, Gpep, ston, KLAY ... ) converting functions & its examples
- Update AccountStore's refresh function for input parameters (Wallet, [addr, privatekey], [privatekey])

### CODE REVIEW

**Comment by kjeom** [comment]
@nohkwak If you have done, please remove the WIP mark

File: ethers-ext/src/core/util.ts [hunk 1]
```diff
@@ -43,3 +43,175 @@ export const HexStr = {
...
+  { unit: 'KLAY', pebFactor: 18 },
+  { unit: 'kKLAY', pebFactor: 21 },
+  { unit: 'MKLAY', pebFactor: 24 },
+  { unit: 'GKLAY', pebFactor: 27 },
+  { unit: 'TKLAY', pebFactor: 30 },
+];
+
+function getFactor(unit: string): number {
+  for ( let i=0; i < KlayUnit.length ; i++ ) {
+    if ( KlayUnit[i].unit === unit) {
```
Comment (blukat29) [inline ethers-ext/src/core/util.ts:118]: toLowerCase to both sides please.

File: ethers-ext/src/core/util.ts [hunk 2]
```diff
@@ -43,3 +43,175 @@ export const HexStr = {
...
+  }
+
+  // @ts-ignore
+  return FixedNumber.fromString(value, { decimals, width: 512 }).value;
+}
+
+/**
+*  Converts %%value%% into a //decimal string// using 18 decimal places.
+*/
+export function formatKlaytn(peb: BigNumberish): string {
```
Comment (blukat29) [inline ethers-ext/src/core/util.ts]: formatKlay and parseKlay
because we're handling the KLAY currency.

File: ethers-ext/src/core/util.ts [hunk 3]
```diff
@@ -43,3 +43,175 @@ export const HexStr = {
     return ethers.utils.hexZeroPad( value, length);
   }
 };
+
+// Klaytn Type Enumeration
+export enum Klaytn {
```
Comment (blukat29) [inline ethers-ext/src/core/util.ts:48]: What about having two enums?
- enum TxType { ValueTransfer = 8, ... }
- enum AccountKeyType { Legacy = 1, ... } 

"Klaytn" is way too generic.

File: ethers-ext/example/transactions/Basic_10_TxTypeValueTransferMemo.js [hunk 4]
```diff
@@ -17,7 +15,7 @@ async function main() {
   const wallet = new Wallet(senderPriv, provider);
 
   tx = {
-      type: 0x10,         
+      type: Klaytn.TxTypeValueTransferMemo,         
       to: recieverAddr,
       value: 1e12,
```
[the remark this answers is not in the document]
Comment (nohkwak) [reply ethers-ext/example/transactions/Basic_10_TxTypeValueTransferMemo.js:20]: I will update.
( There are many examples, so it takes quite a time to check & update. )

File: ethers-ext/src/core/util.ts (hunk 2 above)
Comment (nohkwak) [reply ethers-ext/src/core/util.ts]: Thanks. 
I also found my mistake and already changed.

File: ethers-ext/src/core/util.ts (hunk 1 above)
Comment (nohkwak) [reply ethers-ext/src/core/util.ts:117]: But LowerCase comparison arises some problem. 
In Klaytn specification, we can not distinguish between mKLAY and MKLAY. 
https://docs.klaytn.foundation/content/klaytn/design/klaytn-native-coin-klay

File: ethers-ext/src/core/util.ts (hunk 3 above)
Comment (nohkwak) [reply ethers-ext/src/core/util.ts:48]: How about not using enum (3rd suggestion without '.' ) ? 

e.g. 
1. now :  Klaytn.TxTypeValueTransfer 
2. your suggestion :  TxType**.**ValueTransfer 
3. another suggestion :  TxTypeValueTransfer

File: ethers-ext/src/core/util.ts (hunk 1 above)
Comment (nohkwak) [reply ethers-ext/src/core/util.ts:117]: I will try to add toLowerCase to both sides + comparing 1st characters.

File: ethers-ext/src/core/util.ts (hunk 3 above)
Comment (kjeom) [reply ethers-ext/src/core/util.ts:48]: @nohkwak Is it possible to import a single line when we want to import several tx types?

Comment (nohkwak) [reply ethers-ext/src/core/util.ts]: Good point. 
So I updated 2nd suggestion(https://github.com/klaytn/web3klaytn/pull/440/commits/3e9d9a1f6d2956d6c67d90ae9ff410232e9ee2ea).

File: ethers-ext/src/core/util.ts (hunk 1 above)
Comment (blukat29) [reply ethers-ext/src/core/util.ts]: I see the ambiguity. Thanks.

File: ethers-ext/example/transactions/Basic_10_TxTypeValueTransferMemo.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (nohkwak) [reply ethers-ext/example/transactions/Basic_10_TxTypeValueTransferMemo.js]: Updated examples (https://github.com/klaytn/web3klaytn/pull/440/commits/49d18d6c5d214968ff092bc51b2d78f53e59d9fd).

**Review by blukat29 — APPROVED** [review]

**Review by kjeom — APPROVED** [review]

**Pull request merged**